### ISSUE TITLE
Add support for ksh

### ISSUE THREAD

**Issue by Tagar** [ContinuumIO/anaconda-issues#10027]
```
$ source activate py27_rd
Only bash and zsh are supported


```

Would be great to add support for `ksh` as it is a default shell interpreter in our organization.

**Comment by nehaljwani** [comment]
Related: https://github.com/conda/conda/issues/6820

**Comment by Tagar** [comment]
good point @guziy - it might be an easy fix just changing `local` to `typeset` to add support for ksh
